src/python/WMComponent/DBS3Buffer/DBSUploadPoller.py

```diff
@@ -191,7 +196,8 @@ def __init__(self, config):
         self.datasetType = getattr(self.config.DBS3Upload, "datasetType", "PRODUCTION")
         self.primaryDatasetType = getattr(self.config.DBS3Upload, "primaryDatasetType", "mc")
         self.blockCount = 0
-        self.dbsApi = DbsApi(url=self.dbsUrl)
+        self.gzipEncoding = getattr(self.config.DBS3Upload, 'gzipEncoding', False)
+        self.dbsApi = DbsApi(url=self.dbsUrl, useGzip=self.gzipEncoding)
```

Comment:
@vkuznet Valentin, it looks like we do not need to use gzip when getting this instance. It's only used for checking whether a block exists in the DBS server. I'd say not using it for this use case will reduce the overall latency for such checks (no need to compress before and after send).